### PR TITLE
nfs-ganesha: update cmake options for rpms

### PR DESCRIPTION
**Pull request by alimaredia**
Signed-off-by: Ali Maredia <amaredia@redhat.com>

### CODE REVIEW

**Comment by alimaredia** [comment]
@ajarr let me know if I need to add anything else.

File: nfs-ganesha-stable/build/build_rpm [hunk 1]
```diff
@@ -67,7 +67,7 @@ mkdir build
 cd build
 
 # generate .spec file, edit .spec file for correct versions of libs and make source tarball
-cmake -DCMAKE_BUILD_TYPE=Maintainer -DUSE_FSAL_GLUSTER=OFF -DUSE_FSAL_CEPH=ON -DUSE_FSAL_RGW=ON $WORKSPACE/nfs-ganesha/src && make dist || exit 1
+cmake -DCMAKE_BUILD_TYPE=Maintainer -DUSE_FSAL_GLUSTER=OFF -DUSE_FSAL_CEPH=ON -DUSE_FSAL_RGW=ON -DRADOS_URLS=ON -DUSE_RADOS_RECOV=ON $WORKSPACE/nfs-ganesha/src && make dist || exit 1
```
Comment (ajarr) [inline nfs-ganesha-stable/build/build_rpm]: Stable build is based on v2.5, right? The RADOS based features were introduced in v2.6.  So don't need this change?

Comment (alimaredia) [reply nfs-ganesha-stable/build/build_rpm]: right, thanks

**Comment by alimaredia** [comment]
@ajarr it should be good to go now.

**Pull request merged**